### DISCUSSION TITLE
(Parte 2) Como leer los paquetes

### DISCUSSION THREAD

**Discussion by Cristian1754** [IIC2133-PUC/2024-1#147] [Tarea 2] [answered]
Buenas tardes, me surge la dudad de como debo leer los paquetes en el codigo base de la parte 2. Ya que no hay ninguna variable que los guarde (solo existen dentro del for) ni siquiera se les pide un calloc. Debo crear una estructura que vaya guardando su información?. o que es lo que espera que haga?

**Comment by sebacarrascop** [comment, marked as answer]
hola!, eso queda a tu criterio, se da un código base que es capaz de imprimir ciertas cosas, es tu decisión si deseas utilizar lo que ya entregamos para avanzar o borrarlo e implementarlo a tu gusto.
